src/events/message/index.ts

```diff
@@ -28,42 +30,40 @@ export default class MessageEvent extends Event {
 				return;
 			}
 
-			const fileExtension = extname(file.name).replace(".", "");
-			const language = fileExtension === "" ? "txt" : fileExtension;
+			const fileExtension = extname(file.name).substring(1);
+			const language = fileExtension || "txt";
 			if (language !== "txt" && !extensions.has(language)) {
 				return;
 			}
```

Comment:
You could just forward the file in a multipart request, all the langage/extension is done by the backend already. See https://github.com/readthedocs-fr/bin/blob/main/bin/controller.py#L39-L42

Reply:
On fais ca pour ignorer le message si c'est pas du code (par exemple pour pas tomber sur des images ou autres)

Reply:
Effectivement, il faudrait qu'on se blinde contre ça aussi dans le backend, qu'on soit pas un service d'hébergement d'image (via /raw), tu veux bien ouvrir une issue ?

Reply:
Yep

Reply:
Heu, quand j'y pense, c'est pas vraiment possible d'héberger une image sur rtdbin si ?